crates/spk-cli/cmd-du/src/cmd_du_test.rs

```
@@ -123,40 +121,6 @@ async fn test_du_trivially_works(#[case] solver: SolverImpl) {
     assert_eq!(expected_output.len(), 0);
 }
 
-#[tokio::test]
-async fn test_du_warnings_when_object_is_tree_or_blob() {
```

Why is this test being deleted?

---

The original du command code returned an error if it encountered an spfs blob object. The new du command code both sets ignore errors on the repo walker, and the walker skips blobs entirely and uses spfs entries for files from manifests fir data. The error this was testing for is no longer generated, so the test isn't needed anymore.